AnnoyancesFilter/sections/popups.txt

```diff
@@ -1451,6 +1451,10 @@ radioego.com#%#document.cookie = "facebook_like_page_2=shown; path=/;";
 accorhotelsarena.com#%#document.cookie = "sessionBercy=1; path=/;";
 accorhotelsarena.com#$##block-popb-popin-popinnewsletter { display: none!important; }
 accorhotelsarena.com#$#body[style="overflow: hidden;"] { overflow: visible!important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/31108
+!+ PLATFORM(ios, android)
```

Comment:
iOS does not support #$# rules. When I delete it from code I can still browse on my Android. We we need #$# rules here or are normal ones sufficient?

Reply:
Good point. Now I try only this rule `eapteka.ru##.mobile-fullscreen-banner` and can browse.
So, I think not need to use css rule.